src/app/dashboard/flight-table/flight-table.component.html

```diff
@@ -57,15 +57,15 @@
       <th mat-header-cell *matHeaderCellDef mat-sort-header>
```

Comment:
Ran a prettier format on save here to keep prettier's opinionated formatting in this file.